addons/csw/initSettings.sqf

```diff
@@ -20,6 +20,16 @@ private _categoryArray = [format ["ACE %1", localize LSTRING(DisplayName)]];
     true // Needs mission restart
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(handleExtraMagazinesType), "LIST",
```

Comment:
we could merge this with the `handleExtraMagazines` setting
0 - off
1 - ground
2 - box

but I think it's fine as is

Reply:
I'd rather not invalidate the current setting.